Gruntfile.js

```diff
@@ -78,6 +78,10 @@ module.exports = function (grunt) {
 			dest: '<%= dirs.dest %>/ably-commonjs.js',
 			nonull: true
 		},
+        'ably-nativescript': {
```

Comment:
Tabs pls, unless editing an existing file that doesn't use tabs (eg `package.json`)